sessionctx/variable/BUILD.bazel

```diff
@@ -41,6 +41,7 @@ go_library(
         "//util/collate",
         "//util/dbterror",
         "//util/execdetails",
+        "//util/kvcache",
```

Comment:
Why 'kvcache' and 'planner/core' are added?

Reply:
#37028 add those dependencies. but he forgets to update.

BTW, I think we should add a blocklist to avoid unnecessary dependencies. it looks like this.

https://github.com/pingcap/tidb/commit/e3ceffa580f02fd18b43e4e7d03c76bd5b7b408c

https://github.com/pingcap/tidb/blob/e3ceffa580f02fd18b43e4e7d03c76bd5b7b408c/ddl/BUILD.bazel#L3-L9

Reply:
it means that ```ddl``` is  invisible to any package in the ```planner```